logrotation/test/image/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM centos:7.9.2009
+
+# Installing required rpm and python pkg
+RUN yum install -y epel-release
+RUN yum install -y gcc rpm-build python36 python36-pip \
+    python36-devel python36-setuptools openssl-devel libffi-devel \
+    python36-dbus crontabs logrotate
+
+COPY ./test/py-utils/ /tmp/
+
+RUN pip3 install -r /tmp/python_requirements.txt
+RUN pip3 install -r /tmp/python_requirements.ext.txt
```

Comment:
cortx-prereq pkg wont suffice ?

Reply:
All the things that I have placed under /test/ dir will not be there with components or in actual implementation Chetan, It is for my testing purpose.
All this installation will already there with cortx-all image.

Reply:
alright. so you didnt test with cortx-all image